CONTRIBUTING.md

```diff
@@ -0,0 +1,256 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+  
+You can contribute in one of the following ways:
+
+* Submit a bug report or a feature
+   request in [ our public Jira ](https://jira.percona.com/projects/PS/issues).
+* Submit a pull request (PR) with the code patch.
+* Contribute to [the documentation](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md)
+* Reach us on our [ Forums ](https://forums.percona.com/c/mysql-mariadb)
+
+This document describes the workflow for submitting pull requests.
```

Comment:
The common document structure is often called the "Tell, Show, Tell" approach, as it involves telling the reader what to expect, providing the details, and then reminding them of the key points. It's a helpful framework for organizing information and ensuring your message is clear and effective.

Reply:
Steps added